subxt/src/rpc.rs

```diff
@@ -350,6 +361,22 @@ impl<T: Runtime> Rpc<T> {
             .map_err(Into::into)
     }
 
+    /// Query storage changes
+    pub async fn get_storage_changes(
```

Comment:
Just wonder if it's possible to add the rpc in `phost` rather than subxt? Not quite a problem, but if we can define it outside subxt, we can make it easier to maintain.

Reply:
> Just wonder if it's possible to add the rpc in `phost` rather than subxt? Not quite a problem, but if we can define it outside subxt, we can make it easier to maintain.

I thought the same with you. Just found that the `.client` is priviate which prevent us to access it outside.